File: entry_point/python_entry_point/entry_point.py
import sys
#import sys, re, os

def function():
  print("function call")

def main():
  if len(sys.argv) != 2:
    print("Usage: %s <arg1>" % sys.argv[0])
    sys.exit(1)
  function()
  sys.exit(), end

main()

File: entry_point/c_entry_point/entry_point.c
#include <stdio.h>
#include <stdlib.h>

int main( int argc, char *argv[] ) {
  if( argc != 2 ) {
    fprintf( stderr, "Usage: %s <arg1>\n", argv[0] );
    exit( 1 );
  }
  printf( "%s\n", argv[1] );

  if( argc != 1 ) {
    fprintf( stderr, "Usage: %s <noargs>\n", argv[0] );
    exit( 1 );
  }

  return 0;
}
